test/typings/legacy-component-aliases.types.ts

```diff
@@ -60,6 +60,7 @@ import { NotificationElement } from '@vaadin/vaadin-notification';
 import { HorizontalLayoutElement } from '@vaadin/vaadin-ordered-layout';
 import { VerticalLayoutElement } from '@vaadin/vaadin-ordered-layout';
 import { ScrollerElement } from '@vaadin/vaadin-ordered-layout';
+import { OverlayElement } from '@vaadin/vaadin-overlay';
```

Comment:
I can't see it used anywhere. Did you miss adding an assertion for it?

Reply:
Thanks, added.